filters/encrypt/map.go

```diff
@@ -0,0 +1,26 @@
+package encrypt
+
+// PointerTag provides the pointerstructure pointer string to get/set a key
+// within a map[string]interface{} along with its DataClassification and
+// FilterOperation.
+type PointerTag struct {
+
```

Comment:
Hmm, so it seems `gofmt` is ok with this leading white space if there is a comment following.  It does not enforce it, therefore leaving it to the dev to decide.  
Perhaps the white spaces increases readability of the comment, not sure...
I would argue it should be removed because in almost all other structs we do not have leading white space, but I am ok with keeping it...if you do remove it, `Taggable` below also has leading white space